components/o-topper/src/scss/_mixins.scss

```
@@ -345,6 +345,9 @@
 				color: oColorsMix($foreground, $color-name, 73);
 			}
 		}
+		:focus-visible {
+			outline: 2px solid oColorsByName('white');
```

It would be better to be more specific so we can control other properties of the focus style independently.

```suggestion
			outline-color: oColorsByName('white');
```

Even better, in the future we'll be able to control this by setting a css custom property :)